ide/static/js/layer.js

```diff
@@ -43,6 +43,25 @@ class Layer extends React.Component {
     let comments = [];
     let addCommentModal = null;
     let commentButton = null;
+    let highlightUser = null;
+    let highlightClass = '';
+    let highlightColor = '#000';
+
+    if(this.props.layer.highlight && this.props.layer.highlight.length > 0) {
+      highlightClass = 'highlighted';
+      highlightColor = this.props.layer.highlightColor[this.props.layer.highlightColor.length - 1];
+      highlightUser = (<div style={{
+                              background: highlightColor,
+                              color: 'white',
```

Comment:
Can we shift these style specifications to the css stylesheet as they are fixed?